zp-relayer/index.ts

```diff
@@ -8,6 +8,8 @@ import { init } from './init'
 init().then(() => {
   const app = express()
 
+  app.set('trust proxy', true)
```

Comment:
If we are not using the proxy ourselves, then anyone would be able to mimic a proxy and fake theirs IP, let's make this configurable via a feature flag.

Reply:
> If we are not using the proxy ourselves, then anyone would be able to mimic a proxy and fake theirs IP

Yes. Thought about making a configurable whitelist of trusted IPs to prevent it

Reply:
Also possible, list of all Cloudflare IPs is well-known - https://www.cloudflare.com/ips/

Reply:
> list of all Cloudflare IPs is well-known

Do we use Cloudflare with productions setup?

Reply:
Yes, we do

Reply:
Thought that maintaining a list of trusted IPs for staging/prod would be too cumbersome, so just added a simple flag as you suggested.